cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -366,6 +366,17 @@ protected HttpResponse addArtifact(Id.Artifact artifactId, InputStream artifactC
     return execute(request);
   }
 
+  // add artifact properties and return the response code
+  protected HttpResponse addArtifactProperties(Id.Artifact artifactId,
+                                               Map<String, String> properties) throws Exception {
+    String nonNamespacePath = String.format("artifacts/%s/versions/%s/properties",
```

Comment:
yeah, but its okay, we don't have tests for deleting properties for Artifact handler.